api/Event.json

```
@@ -1022,10 +1022,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "3"
```

`srcElement` is in https://github.com/WebKit/WebKit/blob/b923c2f4eb7804aae65d67b4c3061d6435acb10d/WebCore/khtml/ecma/kjs_events.cpp#L183

```suggestion
              "version_added": "1"
```